apstools/utils.py

```diff
@@ -727,6 +729,60 @@ def split_quoted_line(line):
     return parts
 
 
+def summarize_runs(since=None):
+    """
+    Report bluesky run metrics from the databroker.
+
+    * How many different plans?
+    * How many runs?
+    * How many times each run was used?
+    * How frequently?  (TODO:)
+
+    PARAMETERS
+
+    since (str) :
+        Report all runs since this ISO8601 date & time (default: ``1995``)
+    """
+    since = since or "1995"
+    cat = db.v2.search(databroker.queries.TimeRange(since=since))
+    plans = defaultdict(list)
+    t0 = time.time()
+    for n, uid in enumerate(cat):
+        t1 = time.time()
```

Comment:
Codacy found an issue: [local variable 't1' is assigned to but never used (F841)](https://app.codacy.com/gh/BCDA-APS/apstools/pullRequest?prid=5832153)